src/components/Form.js

```diff
@@ -0,0 +1,62 @@
+import { useState } from 'react'
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+import React from 'react'
+import Question from './Question'
+
+Form.propTypes = {
+  questions: PropTypes.arrayOf(
+    PropTypes.shape({
+      question: PropTypes.string,
+      options: PropTypes.arrayOf(PropTypes.string),
+      id: PropTypes.number,
+    })
+  ).isRequired,
+  onSaveHouse: PropTypes.func.isRequired,
+}
+
+export default function Form({ questions, onSaveHouse }) {
+  const [formData, setFormData] = useState({})
+  const [steps, setSteps] = useState(0)
+
+  const totalSteps = questions.length - 1
+
+  return (
+    <FormStyled onSubmit={handleSubmit}>
+      {questions.map(({ question, options, id }, index) => (
```

Comment:
Imho the `Question` component is a little bit overloaded. I would pull the logic for displaying the correct question up into this Form. Could be as simple as 
```javascript
const currentQuestion = questions[currentStep]
```

Reply:
Thanks! I will have a look 🙂

Reply:
Such a good recommendation! Thank you! 👍